### DISCUSSION TITLE
error CS1537: The using alias  'Exception' appeared

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#25513] [General]
Hello,

I am trying to build/run .net8.0-android project using VSCode (with .net maui extension). When I try to run my project I get the following error

```
Including assemblies for Hot Reload support
D:\Projects\Personal\MyProj\src\MyProj.AndroidTV\obj\Debug\net8.0-android\Microsoft.VisualStudio.HotReload.MobileTapInitializer\Micr
osoft.VisualStudio.HotReload.MobileTapInitializer.ModuleInitializerGenerator\HotReloadMobileTapInitializer.g.cs(13,7): error CS1537: The using alias
 'Exception' appeared previously in this namespace [D:\Projects\Personal\MyProj\src\MyProj.AndroidTV\MyProj.AndroidTV.csproj]
```

Following is the dotnet command it's trying to run... I am getting same error running the command from dotnet cli

```
dotnet build -t:Run -p:Configuration=Debug -p:AdbTarget=-s%20emulator-5554 -p:AndroidAttachDebugger=true -p:AndroidSdbTargetPort=60739 -p:AndroidSdbHostPort=60739 -p:CustomAfterMicrosoftCSharpTargets="c:\Users\<USERNAME>\.vscode\extensions\ms-dotnettools.dotnet-maui-1.4.36-win32-x64\dist\resources\Custom.After.Microsoft.CSharp.targets" -p:MauiVSCodeBuildOutputFile="C:\Users\<USERNAME>\AppData\Local\Temp\dotnet-maui\maui-vsc-2cf77186-352e-4d94-9424-4f6b1360c88a.json" '-p:JavaSdkDirectory="C:\Program Files (x86)\Android\openjdk\jdk-17.0.13.11-hotspot"' -p:XamlTools="c:\Users\<USERNAME>\.vscode\extensions\ms-dotnettools.csharp-2.50.27-win32-x64\.xamlTools" D:\Projects\MyProj\src\MyProj.AndroidTV\MyProj.AndroidTV.csproj 
```
.Net Version : 8.0.403

Any idea what could be the issue?

**Comment by matt-goldman** [comment]
I'm getting errors with this file too (`HotReloadMobileTapInitializer`) but not the using alias error you're getting. I wonder if there's a problem here.

@isaranghi do you have the .NET 9 RC or preview installed by any chance?

**Reply by matt-goldman** [reply]
Weirdly there's a similar error reported in the C# DevKit extension for VS Codde: https://github.com/microsoft/vscode-dotnettools/issues/1247

But that was resolved, and I'm using Visual Studio 🤷

**Reply by matt-goldman** [reply]
I just switched to VS Code and had the same problem. Switched the C# DevKit to pre-release version (I already had the .NET MAUI extension set to pre-release), and I can now run from VS Code. But I still can't run from Visual Studio. Might try a clean install.
